src/index.html

```diff
@@ -9,6 +9,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+         src="images/logo.png"
+         alt="moyo_logo"
+         class="logo_img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link lapcom"
+            href="#laptops"
+            data-qa="hover"
+            >
```

Comment:
Need to fix 
<img width="1061" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/d911223b-6945-4d58-82ec-02efac95a20e">

Reply:
done